src/API/WesternStatesWater.WestDaat.Managers/SystemManager.cs

```diff
@@ -7,64 +7,25 @@
 
 namespace WesternStatesWater.WestDaat.Managers
 {
-    public sealed class SystemManager : ManagerBase, ISystemManager
+    public sealed class SystemManager(
```

Comment:
These primary class constructors are gonna create private fields without underscore prefixes, which I think is fine if we're consistent, but if we don't change the existing `_field` names then we're gonna be split. Thoughts?

Reply:
I know there are some cons to primary class constructors so if you want to keep the `_` prefix for private naming standard we should not use primary class constructors.